lib/src/main/java/io/ably/lib/types/BaseMessage.java

```diff
@@ -81,7 +81,7 @@ public void decode(ChannelOptions opts) throws AblyException {
 						try {
 							data = Base64Coder.decode((String) data);
 						}
-						catch (IllegalArgumentException e) { throw AblyException.fromThrowable(e); }
+						catch (IllegalArgumentException e) { break; }
```

Comment:
Sorry, I should have been clearer. I think we need a log message (`Log.e()` I guess) in this case as well.